### PR TITLE
feat: add Alias to Variance destination doc

### PR DESCRIPTION
**Pull request by gmathieu**
## Description of the change

Now that we support [alias](https://www.variance.com/changelog/updated-list-views#:~:text=Alias,the%20new%20ID.), here's an update to the docs calling it out.

## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

NA

### Development

- [X] Lint rules pass locally
- [X] The code changed/added as part of this pull request has been covered with tests
- [X] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: docs/destinations/crm/variance.mdx [hunk 1]
```diff
@@ -130,6 +130,22 @@ For more information on the <code class="inline-code">group</code> call, refer t
 
 </div>
 
+## Alias
+
+Many destination platforms need an explicit alias call for mapping the already identified users to a new identifier that you may want to use, to track them in the future. The alias call lets you implement this functionality.
```
Comment (ameypv-rudder) [inline docs/destinations/crm/variance.mdx:135]:
```suggestion
Many destination platforms need an explicit `alias` call for mapping the already identified users to a new user identifier that you may want to use, to track them in the future. The [`alias`](https://www.rudderstack.com/docs/rudderstack-api/api-specification/rudderstack-spec/alias/) call lets you implement this functionality.
```

File: docs/destinations/crm/variance.mdx [hunk 2]
```diff
@@ -130,6 +130,22 @@ For more information on the <code class="inline-code">group</code> call, refer t
...
+
+A sample `alias` call is as shown:
+
+```javascript
+rudderanalytics.alias("new_user_id");
+```
+
+<div class="infoBlock">
```
Comment (ameypv-rudder) [inline docs/destinations/crm/variance.mdx:143]:
```suggestion

```

File: docs/destinations/crm/variance.mdx [hunk 3]
```diff
@@ -130,6 +130,22 @@ For more information on the <code class="inline-code">group</code> call, refer t
...
+
+For more information on the <code class="inline-code">alias</code> call, refer to the <a href="https://rudderstack.com/docs/rudderstack-api/http-api">RudderStack HTTP API Specification</a> guide.
```
Comment (ameypv-rudder) [inline docs/destinations/crm/variance.mdx:145]:
```suggestion

```

**Review by ishachopra0401 — APPROVED** [review]

**Comment by gmathieu** [comment]
Thanks for the approval @ishachopra0401. @ameypv-rudder I've addressed your feedback and applied your suggestions to the rest of the doc. Mind taking another look?

**Review by ameypv-rudder — APPROVED** [review]

**Comment by ameypv-rudder** [comment]
Thanks for incorporating the suggestions, @gmathieu - I've approved the PR.

**Comment by gmathieu** [comment]
Thank you! Who's responsible for merging?

**Review by utsabc — APPROVED** [review]

**Comment by ameypv-rudder** [comment]
@gmathieu thanks for all the help :)

**Pull request merged**